### PR TITLE
Edited Readme and fixed a lag issue

### PR DESCRIPTION
**Pull request by Vlammar**
Updated ReadMe file and added a fix for a  lag issue reported in #221 Quick test with spark show that the getEntities() is no longer a lag source. Waiting for reporter (Eris on discord) returns to see if this has solved the issue.

Also fixed two issues concerning wrong map consuming when on survival and map part disapearing when placed

### CODE REVIEW

**Comment by Vulpicula** [comment]
As said over discord, I can confirm that the plugin is now running much lighter using the spark profiler.

File: README.md [hunk 1]
```diff
@@ -209,6 +209,29 @@ This version fixes some small console spam bugs and improves the performance of
 - Updates to [QuartzLib 0.0.4](https://github.com/zDevelopers/QuartzLib/blob/master/CHANGELOG.md#quartzlib-004),
   where all these improvements were made.
 
+### 4.2.0 —The permission and cliffhanger part 1 Update
+Permission and cliffhanger Update finally allow a full official support of minecraft 1.17 and java 17. This comes with bugfixes and new features. We are working on the next big update that will be the 5.0. If you want to know more about this checkout our 5.0 milestone.
```
Comment (DeJayDev) [inline README.md:213]: Maybe link the Milestone? :p

File: README.md [hunk 2]
```diff
@@ -209,6 +209,29 @@ This version fixes some small console spam bugs and improves the performance of
...
+
+This version is only compatible with Minecraft 1.15+. We have no clue if this will work with minecraft 1.18 (it should, but we are not sure)
```
Comment (DeJayDev) [inline README.md:215]: We could also consider dropping official support for 1.15?

bStats reports that only ~1000 servers are still this far behind.

File: README.md [hunk 3]
```diff
@@ -209,6 +209,29 @@ This version fixes some small console spam bugs and improves the performance of
...
+This was tested on Paper 1.17, 1.16 and spigot 1.17,1.15. If there are any errors on a 1.15+ version of spigot/paper please contact us on our discord or create a ticket on the issues page (better)
+
+- 1.17 support
+- Permissions to limit the number of map/image used/owned is now possible. (permissions imageonmap.mapLimit.XX and imageonmap.imageLimit.XX where XX is an integer and will define the limit allowed for the player)
```
Comment (DeJayDev) [inline README.md:220]: Add new line to be consistent.

**Review by DeJayDev — CHANGES_REQUESTED** [review]
I think it could be beneficial if we:
1. Considered dropping official support for 1.15, which will allow QuartzLib to become more robust without worrying about super old versions and their differences.
2. Held off on releasing 4.2.1 until we can also confirm it works with the 1.18 Prereleases (or maybe even the full release)

File: README.md (hunk 2 above)
Comment (Vlammar) [reply README.md:215]: I initially planed to drop 1.15 with the 5.0.0. This should involve map format migration and lot of new things like dithering, interractive maps, "gif" map, optimization and some sort of public map player store on a server...

**Review by DeJayDev — APPROVED** [review]

**Pull request merged**